### PR TITLE
Adding notes about Dartium going away. This is not comprehensive.

### PR DESCRIPTION
**Pull request by Sfshaza**
Fixes: https://github.com/dart-lang/site-webdev/issues/881

Staged:
https://sz-webdev-1.firebaseapp.com/tools
https://sz-webdev-1.firebaseapp.com/tools/dartium
https://sz-webdev-1.firebaseapp.com/tools/webstorm

### CODE REVIEW

File: src/tools/index.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ For information about general tools for Dart apps, see
...
+<aside class="alert alert-info" markdown="1">
+**Dartium is going away in Dart 2.0:**
+In Dart 2.0, you'll use Chrome or other standard browsers for testing
+instead of Dartium, thanks to a new development compiler called
+[_dartdevc_](/tools/dartdevc). For information on Dartium's removal,
+see [A stronger Dart for
+everyone.](http://news.dartlang.org/2017/06/a-stronger-dart-for-everyone.html)
+Once your code is [strong mode]({{site.dartlang}}/guides/language/sound-dart)
+clean, you can start to play with dartdevc now,
+though it doesn't yet work for AngularDart apps.
```
Comment (kwalrath) [inline src/tools/index.md]: If this is going to be exactly like the note above, we should make it an include.

File: src/tools/webstorm.md [hunk 2]
```diff
@@ -21,6 +21,22 @@ with another JetBrains IDE such as IntelliJ IDEA, see the page
...
 
+<aside class="alert alert-info" markdown="1">
+**Dartium is going away in Dart 2.0:**
+In Dart 2.0, you'll use Chrome or other standard browsers for testing
+instead of Dartium, thanks to a new development compiler called
+[_dartdevc_](/tools/dartdevc). For information on Dartium's removal,
+see [A stronger Dart for
+everyone.](http://news.dartlang.org/2017/06/a-stronger-dart-for-everyone.html)
+Once your code is [strong mode]({{site.dartlang}}/guides/language/sound-dart)
+clean, you can start to play with dartdevc now,
```
Comment (kwalrath) [inline src/tools/webstorm.md]: ditto

File: src/tools/dartium.md [hunk 3]
```diff
@@ -10,6 +10,22 @@ Dartium is a special build of Chromium that includes the Dart VM.
...
 until you're ready to test on other browsers.
 
+<aside class="alert alert-info" markdown="1">
+**Dartium is going away in Dart 2.0:**
+In Dart 2.0, you'll use Chrome or other standard browsers for testing
+instead of Dartium, thanks to a new development compiler called
+[_dartdevc_](/tools/dartdevc). For information on Dartium's removal,
+see [A stronger Dart for
+everyone.](http://news.dartlang.org/2017/06/a-stronger-dart-for-everyone.html)
+Once your code is [strong mode]({{site.dartlang}}/guides/language/sound-dart)
```
Comment (kwalrath) [inline src/tools/dartium.md]: Maybe we should link to the Dart 2.0 page instead of the blog post and strong mode page.

Comment (Sfshaza) [inline src/tools/dartium.md]: Yes, I also had that thought. But I need it to go live, first.  :) I'll wait for that page to go live before updating this.

File: src/_includes/dartium-2.0.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<aside class="alert alert-info" markdown="1">
+  **Dartium is going away in Dart 2.0:**
+  In Dart 2.0, you'll use Chrome or other standard browsers for testing
+  instead of Dartium, thanks to a new development compiler called
+  [_dartdevc_](/tools/dartdevc). For information on Dartium's removal,
+  see [Dart 2.0 Updates.]({{site.dartlang}}/dart-2.0)
+  Once your code is [strong mode]({{site.dartlang}}/guides/language/sound-dart)
```
Comment (kwalrath) [inline src/_includes/dartium-2.0.html]: I think we can delete this last sentence. I'm not positive of the facts, and they can get the info from the links.

**Pull request merged**